README.md

```diff
@@ -52,6 +52,7 @@ oracle       | `9.0.1`             | [sample](https://raw.githubusercontent.com/
 oracle       | `9.0.4` to current  | [sample](https://raw.githubusercontent.com/KoskiLabs/jdk-wrapper/master/examples/oracle.9.0.4-current.jdkw)
 oracle       | `10.0.1`            | [sample](https://raw.githubusercontent.com/KoskiLabs/jdk-wrapper/master/examples/oracle.10.0.1-current.jdkw)
 zulu         | any available       | [sample](https://raw.githubusercontent.com/KoskiLabs/jdk-wrapper/master/examples/zulu.any.jdkw)
+adopt        | [any of these](https://api.adoptopenjdk.net/v3/info/release_names?release_type=ga&vendor=adoptopenjdk) | [sample openj9](https://raw.githubusercontent.com/KoskiLabs/jdk-wrapper/master/examples/adpt.openj9.11.jdkw), [sample hotspot](https://raw.githubusercontent.com/KoskiLabs/jdk-wrapper/master/examples/adpt.hotspot.8.jdkw)
```

Comment:
nit: can you rename the files and update the links from `adpt` to `adopt` as well?